modules/install-sync-gateway/install-sync-gateway

```diff
@@ -111,17 +113,17 @@ function install_sync_gateway_on_amazon_linux {
   log_info "Installing SyncGateway from $filepath"
   sudo rpm --install "$filepath"
 
-  # These commands sometimes exit with an error for no obvious reason, so we just ignore the error...
+  # These commands sometimes fail and exit with an error for no obvious reason. Moreover, it appears that the
+  # sync_gateway process can't be stopped cleanly using service, so we subsequently use pkill.
```

Comment:
How weird. I searched around and couldn't find much. Only possible thing is that according to these [random docs](https://chadstechnoworks.com/wptech/db/sample_sync_gateway_installation_and_configuration_procedure.html), on CentOS/RedHat/Oracle 7, perhaps we're supposed to use `systemctl stop sync_gateway`?

Reply:
I tried `systemctl stop sync_gateway` and that actually did nothing (even in the separate shell), so I think `service sync_gateway stop` is correct. It just appears that there is a bug in the `rpm` or initd script that causes it to either start the process incorrectly, or stop it incorrectly such that the current process is tied to the new process.